### ISSUE TITLE
Compability for Redstone Arsenal

### ISSUE THREAD

**Issue by 016Nojr** [ProjectJetDev/SimplyJetpacks-2#1]
Would be nice if it used redstone arsenal's flux-infused chestplate in the jetplate recipe instead of the flux-infused chestplate assembly. And using redstone arsenal's fluxed armor plating.

**Comment by ImMorpheus** [comment]
Indeed, however this was a fork for a server with a modpack without redstone arsenal, thus it's really low priority and it won't be done anytime soon.
